packages/babel-types/test/builders/es2015/__snapshots__/templateElement.js.snap

```diff
@@ -36,6 +37,28 @@ exports[`builders es2015 templateElement should validate 5`] = `
 Property raw expected type of string but got undefined"
 `;
 
+exports[`builders es2015 templateElement should validate 6`] = `
+Object {
+  "tail": false,
+  "type": "TemplateElement",
+  "value": Object {
+    "cooked": undefined,
```

Comment:
cooked should be `null` when the raw can not form a valid template element. See also the AST structure for <code>tag&#96;\u&#96;</code>.